src/transformers/models/vision_text_dual_encoder/modeling_tf_vision_text_dual_encoder.py

```diff
@@ -21,10 +21,15 @@
 from typing import Optional, Tuple, Union
 
 import tensorflow as tf
-from tensorflow.keras.layers import Dense
+
+
+try:
+    from tf_keras.layers import Dense
+except (ImportError, ModuleNotFoundError):
+    from keras.layers import Dense
```

Comment:
To be consistent with other modeling files, could we use `from transformers.modeling_tf_utils import keras` and then `keras.layers.Dense`?

Reply:
Done!